reverse.js

```diff
@@ -0,0 +1,3 @@
+const reverseArray = arr => (result = arr.map(i => i).reverse());
+
+const reverseArrayInPlace = arr => arr.reverse();
```

Comment:
As far as I know, the exercise forbids using `Array.prototype.reverse`: https://eloquentjavascript.net/04_data.html#p_+SVCOC/qcN Implement it yourself :P